examples/arithmetic/tests/bindings/test_arithmetic.py

```diff
@@ -20,6 +20,15 @@
 assert sub(4, 2) == 2
 assert sub(8, 4) == 4
 
+assert div(8, 4) == 2
+
+try:
+    div(8, 0)
+    assert(not("Should have panicked when dividing by zero"))
+except:
```

Comment:
This will catch and ignore the `AssertionError` from "should have panicked" above. In this case you can use the `try-else` syntax like this:

```
try:
    div(8, 0)
except:
    pass
else:
    assert False, "Should have panicked"